packages/module/src/Message/CodeBlockMessage/CodeBlockMessage.scss

```diff
@@ -19,14 +19,15 @@
     --pf-v6-c-code-block__header--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
     --pf-v6-c-code-block__header--PaddingInlineStart: var(--pf-t--global--spacer--md);
     --pf-v6-c-code-block__header--PaddingInlineEnd: var(--pf-t--global--spacer--md);
-    color: var(--pf-t--global--text--color--inverse);
+    color: var(--pf-t--color--white); // same in light + dark theme
 
     .pf-v6-c-code-block__actions {
       width: 100%;
     }
 
     .pf-chatbot__message-code-block-language {
       text-transform: uppercase;
+      font-family: var(--pf-t--global--font--family--body);
```

Comment:
This is a tiny fix to bring us slightly more inline with what designs Kayla has in Figma right now. I'll address larger changes separately.